src/mon/OSDMonitor.cc

```diff
@@ -1199,8 +1216,9 @@ void OSDMonitor::encode_pending(MonitorDBStore::TransactionRef t)
   assert(get_last_committed() + 1 == pending_inc.epoch);
   ::encode(pending_inc, bl, features | CEPH_FEATURE_RESERVED);
 
-  dout(20) << " full_crc " << tmp.get_crc()
-	   << " inc_crc " << pending_inc.inc_crc << dendl;
+  dout(20) << __func__
+           << " full_crc " << tmp.get_crc()
+	   << " inc_crc" << pending_inc.inc_crc << dendl;
```

Comment:
and it's not clear whether this one is really misaligned or if it's just github's presentation. Also, you are missing a space between `inc_crc` and the quotation mark.